lard_tests/tests/end-to-end.rs

```diff
@@ -0,0 +1,115 @@
+use std::future::Future;
+
+use test_case::test_case;
+
+pub mod common;
+
+async fn e2e_test_wrapper<T: Future<Output = ()>>(test: T) {
+    let api_server = common::init_api_server().await;
+    let ingestor = common::init_ingestion_server().await;
+
+    tokio::select! {
+        _ = api_server => panic!("API server task terminated first"),
+        _ = ingestor => panic!("Ingestor server task terminated first"),
+        _ = test => {}
+    }
+}
+
+#[test_case(20000, 0; "to closed timeseries")]
+#[test_case(12000, 6; "to open timeseries")]
+#[tokio::test]
+async fn append_data(station_id: i32, expected_rows: usize) {
+    e2e_test_wrapper(async move {
+        let param_id = 211;
+        let obsinn_msg = format!(
+            "kldata/nationalnr={}/type=508/messageid=23
+TA
+20230505014000,20.0
+20230505014100,20.1
+20230505014200,20.0
+20230505014300,20.2
+20230505014400,20.2
+20230505014500,20.1
+",
```

Comment:
can / should this be formated differently? I guess maybe not since we can't have spaces in front of the lines... but then the first line could also have no spaces?

Reply:
That's what a raw Obsinn message looks like, but we can for sure construct it in a better way in the tests

Reply:
I meant more just for readability, like if the first line was not indented like the rest of them?